### DISCUSSION TITLE
Do not enter the edit view after create

### DISCUSSION THREAD

**Discussion by DiesIrae** [keystonejs/keystone#5532] [Feature requests]
In the create view, when clicking on the "Create" button, why redirecting to the edit view? It seems more logical to me to close the create view and come back to the list view instead. Every time I create an object I have to click on the link to come back to the view list.

The behavior I feel the most natural:
- When in the list page, close the create view panel and add the new object to the list if compatible with filter (or simply refresh the whole list to avoid filter/sort edge cases)
- When in the dashboard page, just close the create view panel

**Comment by ChuckJonas** [comment]
Might be worthwhile to make this configurable.

**Reply by DiesIrae** [reply]
Sure, why not! As I grew frustrated I searched for a config, but found none. But I still don't see the case where redirecting to the edit view is better...

**Reply by ChuckJonas** [reply]
maybe in use cases where you typically need to create children record? 

I hope one day keystonejs has support for "display only" views... In which case being able to specify the `onCreateRedirect` would be even more important.

**Reply by DiesIrae** [reply]
You can create children records in the create view...

But evidently having deep configurable redirections would be nice.😎

**Comment by bladey** [comment]
Thanks for the idea @DiesIrae!

**Reply by DiesIrae** [reply]
You're welcome :-)
